### DISCUSSION TITLE
Favorite device to run openHASP?

### DISCUSSION THREAD

**Discussion by fvanroie** [HASwitchPlate/openHASP#354] [Polls]
openHASP supports over 2 dozen hardware combinations. I opened this poll to get a better view of the most *favored* devices to run the firmware. Please select your (preferred) device below. If your hardware combo is not listed, please select `other` and elaborate in the comments. Thanks!

**Comment by fvanroie** [comment]
Thank you for all your votes, this really help to know which devices to focus on!

I moved the poll to Ki-Fi, because they allow far more choices than GitHub polls.
Feel free to[ have a peek](https://ko-fi.com/polls/Which-device-do-you-use-to-run-openHASP-M4M010VJ6Q), you are free to vote again as well! 😄

**Comment by FreeBear-nc** [comment]
Guition ESP32-S3 4848S040 - Went and bought two 'cos they were relatively dirt cheap and fit is a standard EU/UK square back box. Only downside is only one or two spare GPIO pins.

P.S. Can't/won't use Ki-Fi as it is asking for a log in or sign up.
